teamsupport/models.py

```diff
@@ -23,6 +27,31 @@ def __init__(self, client, ticket_id=None, data=None):
                 '(neither given)')
         self.id = self.TicketID
 
+    @classmethod
+    def create(self, client, first_name, email, category, name, description):
+        # We need to associate ticket with Contact, otherwise ticket doesn't
+        # make sense. First, we try to find an existing contact.
+        contacts = client.search_contacts(FirstName=first_name, Email=email)
+        if len(contacts):
+            contact = contacts[0]
+        else:
+            # Otherwise - create new one.
+            contact = client.create_contact(FirstName=first_name, Email=email)
+        contact_id = contact.find('ContactID').text
+
+        data = {
+            'Name': name,
+            'FormCategory': category,
+            'TicketStatusID': TICKET_STATUS_NEW,
+            'TicketTypeID': TICKET_TYPE_SUPPORT,
+            'ContactID': contact_id
+        }
+
+        ticket_xml = client.create_ticket(data)
+        ticket_id = ticket_xml.find('TicketID').text
```

Comment:
I think it makes sense to initialize ticket with xml obtained from `create_ticket`